js/loadTrack.js

```diff
@@ -0,0 +1,24 @@
+$('.trackLink').click(function(e) {
+    var link = e.target.dataset.track
+        
+    if  (link === '1' && $('#track1').length === 0) {
+        $('#container').append($('<div>').load('includes/trackLayout1.html'))
```

Comment:
To get rid of extra div assign the essential class and id stuff to this div (do this dynamically depending on track1/track2) and delete div from inside includes file
eg:
```$('#container').append($('<div id="track1" class="track">').load('includes/trackLayout1.html'))```

Reply:
done